openedx/openedx-platform
# SE-1076 Campus.il Fixes issue encountered when cms worker updates grades

When using persistent grades, any change to the grades profile in Studio results in a grades recalculation.

This PR fixes an exception caused by a missing setting in the cms envs, which prevents the grades recalculating, e.g.: 
```
Grades: SGF.update
...
Cannot grade student 2 in course course-v1:...  because of exception: 'Settings' object has no attribute 'GENERATE_PROFILE_SCORES'
```

**JIRA tickets**: SE-1076

**Sandbox URL**:

* LMS: https://pr20654.sandbox.opencraft.hosting/
* Studio: https://studio.pr20654.sandbox.opencraft.hosting/

**Testing instructions**:

1. Enable persistent grades.
1. Create a course and enroll some users.
1. Modify the grades policy.
   Watch the logs to see that grades are recalculated without error.

**Reviewers**
